modules/op-about-triggers.adoc

```diff
@@ -93,10 +93,12 @@ spec:
 <4> Parameters supplied by the `TriggerBinding` or `EventListerner` resources.
 <5> List of templates that specify the way resources must be created using the parameters received through the `TriggerBinding` or `EventListener` resources.
 
-* The `Trigger` resource connects the `TriggerBinding` and `TriggerTemplate` resources, and this `Trigger` resource is referenced in the `EventListener` specification.
+* The `Trigger` resource combines the `TriggerBinding` and `TriggerTemplate` resources, and the `interceptors` event processor. 
```

Comment:
Yes agree with @VeereshAradhya 

>Line number 93 says Parameters supplied by the 'TriggerBinding' or 'EventListerner' resources. Is it both @savitaashture @khrm

Parameters supplied by the 'TriggerBinding' resources

>interceptors are optional, I think we need to include that in line number 96
ref: https://github.com/tektoncd/triggers/blob/release-v0.14.x/docs/triggers.md

Yes interceptors are optional

>The TriggerBinding resource validates events, I think we should remove validates events. WDYT @savitaashture @khrm

Yes we can remove because `TriggerBinding` resource won't validate events.

Reply:
I have incorporated all the comments.